deployment/testing-v3/compose/admin/.env.template

```diff
@@ -0,0 +1,21 @@
+# This configuration is generated by scripts/generate-dotenv. Do not check in
+# manually edited values into source control.
+
+APP_NAME=test-app
+DATA_CONFIG=/app/config/content.json
+ORIGIN_CONFIG=/app/config/telemetry_origin_data_inc.json
+BUCKET_PREFIX=test-app/v1
+
+# relative to the docker-compose file
+GOOGLE_APPLICATION_CREDENTIALS="../../.service-account-keys/service-account-admin-private-key.json"
+
+PUBLIC_KEY_HEX_INTERNAL=1723D28D922FD045A3EBECE5FA9BBD67DF28B60B5666203DA06E0CE296D7DF11
+BUCKET_INTERNAL_PRIVATE=gs://a-private-a82843a795cf9ef5
```

Comment:
Just confirming that we're ok with checking these into source control?

Reply:
Yep, I wanted to have the .env.template have realistic values to be able to reference. There's nothing sensitive in these configuration files.